### PR TITLE
Rename ADX -> ATP

### PR DESCRIPTION
**Pull request by pfrazee**
- Changes npm org name to `@atproto`
- Update readme
- Update a lot of semantics, including eg `AdxUri` to `AtUri`
- Changed the UCAN semantics (daniel you should review this at some point, let's not hold up the merge for it though)
- Remove docs dir (relocated to atproto website)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,25 @@
-# ADX - Authenticated Data eXperiment
+# @ Protocol (Authenticated Transfer Protocol)
 
-This is a working repository for Bluesky's data protocol which we've termed ADX: the Authenticated Data eXperiment. The "X" stands for "experiment" while the project lives in an early exploratory state.
-
-The `main` branch is in active development and does not yet have documentation. We previously released a demo of our data system.
-
-**You can check it out on the [`data-demo` branch](https://github.com/bluesky-social/adx/tree/data-demo).**
+This is a working repository for the "@ Protocol," aka the Authenticated Tramsfer Protocol, aka ATP.
```
Comment (devinivy) [inline README.md:3]:
```suggestion
This is a working repository for the "@ Protocol," aka the Authenticated Transfer Protocol, aka ATP.
```

File: package.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "name": "adx",
+  "name": "atp",
   "version": "0.0.1",
-  "repository": "git@github.com:bluesky-social/adx.git",
-  "author": "Daniel Holmgren <dtholmgren@gmail.com>",
+  "repository": "git@github.com:bluesky-social/atp.git",
+  "author": "Bluesky PBLLC <hello@blueskyweb.xyz>",
```
Comment (dholms) [inline package.json:5]: 😭 
jkjk

File: README.md (hunk 1 above)
Comment (pfrazee) [reply README.md]: lmao good catch

File: README.md [hunk 3]
```diff
@@ -1,29 +1,25 @@
-# ADX - Authenticated Data eXperiment
+# @ Protocol (Authenticated Transfer Protocol)
```
Comment (dholms) [inline README.md:1]: the big `@` img from the website could look good here. by no means necessary

File: packages/cli/src/commands/setup/init.ts [hunk 4]
```diff
@@ -21,12 +21,12 @@ export default cmd({
...
-███████║██║  ██║ ╚███╔╝
-██╔══██║██║  ██║ ██╔██╗
-██║  ██║██████╔╝██╔╝ ██╗
-╚═╝  ╚═╝╚═════╝ ╚═╝  ╚═╝`)
+ ██████╗ 
+██╔═══██╗
+██║██╗██║
+██║██║██║
+╚█║████╔╝
+ ╚╝╚═══╝ `)
```
Comment (dholms) [inline packages/cli/src/commands/setup/init.ts:29]: hehe nice

**Review by dholms — APPROVED** [review]
💪 😎

**Pull request merged**